src/js/view/controls/settings-menu.js

```diff
@@ -85,13 +97,24 @@ export function setupSubmenuListeners(settingsMenu, controlbar, viewModel, api)
             return;
         }
 
+        const currentLevel = model.get('currentLevel');
+
         addQualitiesSubmenu(
             settingsMenu,
             levels,
             (index) => api.setCurrentQuality(index),
-            model.get('currentLevel'),
+            currentLevel,
             model.get('localization').hd
         );
+
+        if (currentLevel === 0) {
```

Comment:
Why doesn't the "change:currentLevel" or "change:visualQuality" listeners capture this change?